tools/cleanup/cleanup.sh

```diff
@@ -0,0 +1,285 @@
+#!/bin/bash 
+
+# Cleans up all Trilio resources from the k8s cluster
+# IMP - Runs on the sourced kubeconfig (refers to .kube/config)
+# Please make sure that correct kubeconfig is used or sourced
+# Deleted resources can not be recovered
+
+CLEANUP_RUN_SUCCESS=true
+
+check_if_ocp() {
+  # Check if the k8s cluster is upstream or OCP
+  local is_ocp="N"
+  if (kubectl api-resources | grep -q openshift.io); then
+    is_ocp="Y"
+  fi
+  echo "${is_ocp}"
```

Comment:
log it properly

Reply:
This is used as return value, the log is added where this function is called.